experimental/dds/tree2/src/feature-libraries/sequence-field/rebase.ts

```diff
@@ -866,30 +611,55 @@ function tryRemoveLineageEvent<T>(mark: Attach<T>, revisionToRemove: RevisionTag
 	}
 }
 
-/**
- * @returns true iff both reattaches target cells that were affected by the same detach.
- * The target cells may or may not overlap depending on detach index information.
- *
- * Only valid in the context of a rebase (i.e., both marks have the same input context).
- */
-function areRelatedReattaches<T>(baseMark: Reattach<T>, newMark: Reattach<T>): boolean {
-	const baseReference = baseMark.lastDetachedBy ?? baseMark.detachedBy;
-	const newReference = newMark.lastDetachedBy ?? newMark.detachedBy;
-	return baseReference === newReference;
-}
-
-/**
- * @returns true iff `baseMark` attaches nodes in cells whose contents were detached by the same change
- * that conflicts with `newMark`.
- * The target cells may or may not overlap depending on detach index information.
- */
-function isBaseAttachRelatedToConflictedDetach<T>(
-	baseMark: NewAttach<T> | Reattach<T>,
-	newMark: Detach<T> & Conflicted,
+function compareCellPositions(
```

Comment:
Some of the comments left on the `compareCellPositions` of `compose.ts` are likely relevant here.